package.json

```diff
@@ -535,40 +535,42 @@
   "scripts": {
     "vscode:prepublish": "webpack --mode production",
     "compile": "tsc -p ./",
-    "postinstall": "node ./node_modules/vscode/bin/install",
     "tslint": "tslint -t verbose src/**/*.ts",
-    "test": "node ./node_modules/vscode/bin/test",
+    "test": "tsc -p ./ && node ./out/test/runTests.js",
     "webpack": "webpack --mode development",
     "webpack-dev": "webpack --mode development --watch"
   },
   "devDependencies": {
+    "@azure/arm-resources": "^4.2.2",
+    "@azure/arm-resources-subscriptions": "^2.0.0-beta.1",
```

Comment:
>beta [](http://example.com/codeflow?start=50&length=4)

is there a stable version we could use? I worry of taking dependencies on a beta version.

Reply:
There are exactly two releases of this module, 1.0.0-beta.1 or 2.0.0-beta.1 [https://github.com/Azure/azure-sdk-for-js/releases?q=arm-resources-subscriptions&expanded=true](url)